bridge-status-bot/app.sh

```diff
@@ -44,6 +45,8 @@ ETH_BALANCE=$(get_eth_balance "$BRIDGE_ETHEREUM_ADDRESS")
 ETH_BALANCE_THRESHOLD=0.5
 ETH_SAFE_EXECUTOR_BALANCE=$(get_eth_balance "$BRIDGE_ETHEREUM_SAFE_EXECUTOR_ADDRESS")
 ETH_SAFE_EXECUTOR_THRESHOLD=2
+ETH_BRIDGE_BSC_OPERATOR_BALANCE=$(get_eth_balance "$BRIDGE_BSC_OPERATOR_ADDRESS")
+ETH_BRIDGE_BSC_OPERATOR_THRESHOLE=2
```

Comment:
Maybe it seems like typo

```suggestion
ETH_BRIDGE_BSC_OPERATOR_THRESHOLD=2
```

Reply:
Fixed!